containers/src/Data/IntSet/Internal.hs

```diff
@@ -1215,6 +1216,48 @@ fromList xs
   where
     ins t x  = insert x t
 
+-- | \(O(n / W)\). Create a set from a range of integers.
+--
+-- > fromRange (l,h) == fromList [l..h]
```

Comment:
```suggestion
-- | \(O(n / W)\). Create a set from a range of integers.
--
-- > fromRange (low, high) == fromList [low..high]
--
-- If @low > high@, @fromRange (low, high)@ is empty.
```

Reply:
Changed to `low`, `high` 👍 

> `If @low > high@, @fromRange (low, high)@ is empty`

I don't think this adds value though because this is already conveyed by the line above. `[low..high]` is empty if `low` > `high`.

Reply:
> > `If @low > high@, @fromRange (low, high)@ is empty`
> 
> I don't think this adds value though because this is already conveyed by the line above. `[low..high]` is empty if `low` > `high`.

I thought it might be good to point out the special case, even if it's redundant, but I have no strong feelings on this.